client/src/components/Token-Choices.vue

```diff
@@ -42,15 +42,13 @@ export default {
     decrement () {
       if (this.currTokens + this.choiceCost <= this.totalTokens && this.numChoice - 1 >= 0) {
         EventBus.$emit('decrement', this.choiceCost)
-        
+
         this.numChoice--
         this.emitChoiceCount()
       }
     },
     emitChoiceCount () {
-      this.choice == 'pick' ? 
-      EventBus.$emit('numPicks', this.numChoice) :
```

Comment:
Event names need to be kebab case
https://blog.usejournal.com/vue-js-best-practices-c5da8d7af48d